validators/zone_specific_checks.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+import pandas as pd
+
+from electricitymap.contrib.validators.lib.config import validator
+
+
+@validator(
+    kind="production",
+    not_zone_keys=[  # zone_keys where we allow fossil fuel production to be 0
```

Comment:
This array should also include SE as it don't have any fossil fuel production for the vast majority (almost all) of the time.